test/topology_custom/test_tablets.py

```diff
@@ -48,10 +48,10 @@ async def test_tablet_default_initialization(manager: ManagerClient):
     server = await manager.server_add(config=cfg)
 
     cql = manager.get_cql()
-    await cql.run_async("CREATE KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1} AND tablets = {'initial': 1};")
+    await cql.run_async("CREATE KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1} AND tablets = {'enabled': true};")
 
     res = await cql.run_async("SELECT * FROM system_schema.scylla_keyspaces WHERE keyspace_name = 'test'")
-    assert res[0].initial_tablets > 0, "initial_tablets not configured"
+    assert res[0].initial_tablets == 0, "initial_tablets not configured"
```

Comment:
>  It's still not clear why you test for initial_tablets=0 and not initial_tablets>0.

Because if you start with default tablets configuration, it will be 0 initial tablets _in_ _system_schema_ _table_. When a table is created for a keyspace that has 0 initial tablets it will [auto-estimate](https://github.com/scylladb/scylladb/pull/16364#discussion_r1448961336) the real number.